compute/instances_test.go

```diff
@@ -90,6 +90,78 @@ func TestInstanceClient_CreateInstance(t *testing.T) {
 }
 
 // Test that the client can create an instance.
+func TestInstanceClient_CreateInstanceError(t *testing.T) {
+	server := newAuthenticatingServer(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "POST" && r.Method != "GET" && r.Method != "DELETE" {
```

Comment:
We need the delete or it'll error because we now delete an instance when it enters an error state